frame/election-provider-multi-phase/src/benchmarking.rs

```diff
@@ -192,37 +190,22 @@ frame_benchmarking::benchmarks! {
 	}
 
 	on_initialize_open_signed {
-		// NOTE: this benchmark currently doesn't have any components because the length of a db
-		// read/write is not captured. Otherwise, it is quite influenced by how much data
-		// `T::ElectionDataProvider` is reading and passing on.
```

Comment:
this note is still relevant?

Could update benchmarks in the future

Reply:
in the past this benchmark created the snapshot as well, and therefore it was a function of the byte length. Now, it is not doing that anymore, therefore byte length is irrelevant.

Reply:
I get that the removal of these comments seem like mistakes, but they are (hopefully) exactly all signs of reduction of complexity: we no longer need to care about these details in this pallet. Whoever is the implementer of `T::DataProvider` will record its own appropriate weight.